tests/metagpt/test_increment.py

```diff
@@ -20,7 +19,7 @@ def test_refine_simple_calculator():
         "Add subtraction, multiplication and division operations to the calculator. The current calculator can only perform basic addition operations, and it is necessary to introduce subtraction, multiplication, division operation into the calculator",
         "--inc",
         "--project-path",
-        f"{DEFAULT_WORKSPACE_ROOT}/simple_add_calculator",
+        "data/simple_add_calculator",
```

Comment:
1. I mean a const path "DATA_PATH".
2. The packages such as "simple_add_calculator" should be uploaded as zip / tar.gz with related unzip / untar commands.
3. test_increment.py is somewhat unclear, is test_incremental_dev.py better?

Reply:
OK, I will continue to modify the test cases.

Reply:
Modified the key code and uploaded 4 code compression packages (maybe not appropriate, better ones need to be given)